build.gradle

```diff
@@ -445,6 +469,62 @@ integTest {
     systemProperty propertyKeys.breaker.useRealMemory, getBreakerSetting()
 }
 
+task integTestRemoteIndexBuild(type: RestIntegTestTask) {
```

Comment:
Please try to reuse/refactor the other gradle tasks to move the common things at 1 place rather than copying everything.